poetry/core/packages/package.py

```diff
@@ -88,7 +88,8 @@ def __init__(
         self.documentation_url = None
         self.keywords = []
         self._license = None
-        self.readme = None
+        self.readme = ()
```

Comment:
Given the fact that this is now a tuple, we should pluralize it to make clear what is going on.